src/cpp/include/nodegui/QtCore/QColor/qcolor_wrap.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <napi.h>
+#include <stdlib.h>
+
+#include <QColor>
+
+#include "core/Component/component_macro.h"
+
+class QColorWrap : public Napi::ObjectWrap<QColorWrap> {
+ private:
+  std::unique_ptr<QColor> instance;
+
+ public:
+  static Napi::FunctionReference constructor;
+  static Napi::Object init(Napi::Env env, Napi::Object exports);
+  QColorWrap(const Napi::CallbackInfo& info);
+  ~QColorWrap();
+  QColor* getInternalInstance();
+  // Wrapped methods
+
+  COMPONENT_WRAPPED_METHODS_DECLARATION
+};
+
+namespace StaticQColorWrapMethods {
+Napi::Value fromQVariant(const Napi::CallbackInfo& info);
```

Comment:
I think you missed this one.

Reply:
I did, thanks. A lot of copy and pasting I forgot to remove some things.